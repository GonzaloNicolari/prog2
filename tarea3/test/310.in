# prueba de nivel_en_binario
nivel_en_binario 1
insertar_en_binario (1,g)
nivel_en_binario 1
nivel_en_binario 2
insertar_en_binario (2,d)
nivel_en_binario 1
nivel_en_binario 2
nivel_en_binario 3
insertar_en_binario (2,m)
nivel_en_binario 2
insertar_en_binario (3,k)
insertar_en_binario (3,r)
insertar_en_binario (3,b)
insertar_en_binario (3,e)
nivel_en_binario 3
insertar_en_binario (4,a)
insertar_en_binario (4,z)
nivel_en_binario 1
nivel_en_binario 2
nivel_en_binario 3
nivel_en_binario 4
nivel_en_binario 5
Fin