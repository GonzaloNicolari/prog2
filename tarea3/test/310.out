1>#  prueba de nivel_en_binario.
2>
3>Insertado.
4>(1,g)
5>
6>Insertado.
7>(1,g)
8>(2,d)
9>
10>Insertado.
11>(2,d)(2,m)
12>Insertado.
13>Insertado.
14>Insertado.
15>Insertado.
16>(3,b)(3,e)(3,k)(3,r)
17>Insertado.
18>Insertado.
19>(1,g)
20>(2,d)(2,m)
21>(3,b)(3,e)(3,k)(3,r)
22>(4,a)(4,z)
23>
24>Fin.
